<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPicoPlotOffset</class>
 <widget class="QMainWindow" name="FormPicoPlotOffset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XBPM Plot Offsets</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="boxChannel2">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>10</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="boxChannel1">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>10</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="boxChannel4">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>10</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="boxChannel3">
    <property name="geometry">
     <rect>
      <x>286</x>
      <y>10</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 3</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="titleGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>780</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>80</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QEPlot" name="plotChannels">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>780</width>
      <height>530</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEPlot</class>
   <extends>QEFrame</extends>
   <header>QEPlot.h</header>
  </customwidget>
  <customwidget>
   <class>QEFrame</class>
   <extends>QFrame</extends>
   <header>QEFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
